xkcd/main.go

```diff
@@ -0,0 +1,125 @@
+package xkch
```

Comment:
This looks like a typo

```suggestion
package xkcd
```